js/game/Director.js

```diff
@@ -502,4 +502,88 @@ function Director(scene){
+    "coolNoMore"           : "hats ain't cool no more, haven't you heard?"           ,
+    "beScared"             : "BE SCARED. BE ANGRY."                                  ,
+};
+
+// Umlaut codes: Ä = \xC4, Ö = \xD6, Ü = \xDC, ä = \xE4, ö = \xF6, ü = \xFC, ß = \x
+
+var textStrings_DE = {
+    // Act 1
+    "niceHat"              : "Oh, h\xFCbscher Hut!"                                              ,
+    "outtaHere"            : "Ja, besser schnell weg hier"                                       ,
+    "getARoom"             : "Widerlich, nehmt euch ein Zimmer!"                                 ,
+    "notCoolAnymore"       : "Schon geh\xF6rt? H\xFCte sind out."                                ,
+    "tvOnTv"               : "Ein Fernseher! Im Fernsehen!"                                      ,
+    "cricky"               : "Ein Grash\xFCpfer!"                                                ,
+    "tooManyCrickets"      : "Ok, zu viele Grash\xFCpfer."                                       ,
+    "normalPeep"           : "Nur ein gew\xF6hnlicher Typ."                                      ,
+    "normalPeeps"          : "Nur ein paar gew\xF6hnliche Typen."                                ,
+    "wowNothing"           : "Wow! Nichts! Gar nix!"                                             ,
+    
+    // Act 2
+    "crazySquareAttacks"   : "Verr\xFCcktes Quadrat greift an!"                                  ,
+    "justMissed"           : "Ooooh, knapp verpasst."                                            ,
+    "somethingInteresting" : "(du musst sie schon erwischen, wenn was *Interessantes* passiert)" ,
+    "whoIsScreaming"       : "(erwische, *wer* sie anschreit)"                                   ,
+    "circleFearsSquares"   : "KREIS F\xDCRCHTET QUADRATE!"                                       ,
+    "whoScaresThem"        : "(erwische, *wen* sie f\xFCrchten)"                                 ,
+    "areTheyScared"        : "(erwische sie, *wenn* sie fl\xFCchten)"                            ,
+    "squaresSnubCircles"   : "QUADRATE <SNUB> KREISE"                                  ,
+    "areTheySnubbed"       : "(erwische sie, w\xE4hrend sie angreifen)"                          ,
+    "everyoneHates"        : "ALLE HASSEN JEDEN!!1!"                                             ,
+    "almostEveryoneHates"  : "FAST JEDER HASST ALLE..."                                          ,
+    "squaresHateCircles"   : "QUADRATE HASSEN KREISE"                                            ,
+    "circlesHateSquares"   : "KREISE HASSEN QUADRATE"                                            ,
+    "areTheyYelling"       : "(erwische sie, *w\xE4hrend sie sich gegenseitig anschreien)"       ,
+    "nerdsNow"             : "Was machen diese Nerds jetzt?"                                     ,
+    "schockedPeep"         : "Warum ist er geschockt?"                                           ,
```

Comment:
I don't have context for this one - you sure it's not "schockiert"? Or do they mean the same thing?